back/app/controllers/web_api/v1/home_pages_controller.rb

```diff
@@ -31,27 +31,27 @@ def set_home_page
   def home_page_params
     params.require(:home_page).permit(
       :top_info_section_enabled,
-      :top_info_section_multiloc,
       :bottom_info_section_enabled,
-      :bottom_info_section_multiloc,
       :events_widget_enabled,
       :projects_enabled,
-      :projects_header_multiloc,
       :banner_avatars_enabled,
       :customizable_homepage_banner_enabled,
       :banner_layout,
-      :banner_signed_in_header_multiloc,
-      :banner_cta_signed_in_text_multiloc,
       :banner_cta_signed_in_type,
       :banner_cta_signed_in_url,
-      :banner_signed_out_header_multiloc,
-      :banner_signed_out_subheader_multiloc,
       :banner_signed_out_header_overlay_color,
       :banner_signed_out_header_overlay_opacity,
-      :banner_cta_signed_out_text_multiloc,
       :banner_cta_signed_out_type,
       :banner_cta_signed_out_url,
       :header_bg,
+      top_info_section_multiloc: CL2_SUPPORTED_LOCALES,
```

Comment:
I'd like to look at this again with @alexander-cit once he's back to make sure I understand the change

Reply:
So, you allow locales (en, sr-Latn, etc) as keys in the `top_info_section_multiloc` hash with any values. See a bit simpler example here https://apidock.com/rails/ActionController/Parameters/permit

```ruby
params.require(:person).permit(contact: [ :email, :phone ])
```

Let me know if I misunderstood the question or if there's still anything not clear.